src/OrchardCore/OrchardCore.OpenId.Core/YesSql/Indexes/OpenIdAuthorizationIndex.cs

```diff
@@ -8,7 +8,7 @@ public class OpenIdAuthorizationIndex : MapIndex
     {
         public string AuthorizationId { get; set; }
         public string ApplicationId { get; set; }
-        public DateTimeOffset? CreationDate { get; set; }
+        public DateTime? CreationDate { get; set; }
```

Comment:
Here for example if we'd want to keep the offset part we could use a new param/column to store simply the Offset. Should this be needed. Generally for these things a DateTime.UtcDateTime is enough.

Reply:
Yeah, all these dates are used by OpenIddict to ensure things like tokens are not expired. Loosing the offset info is not a big deal here (and that's what the OpenIddict EF 6/EF Core and MongoDB stores do anyway).